### ISSUE TITLE
Make "top websites" more accessible

### ISSUE THREAD

**Issue by dragbeadle** [mozilla-mobile/fenix#12586]
## User Story
Using Firefox Preview / Nightly on Android, there are over half a dozen of websites I visit on a nearly daily basis. I saved these as "top websites".

On the old Firefox for Android I could:

* Tap on the address bar
* Tap on on of these websites. 
Done in two clicks. The UI made perfect use of the empty space between the address bar (top) and keyboard (bottom) and filled those with tiles which could give me access to often used websites with a single tap. 

On previous versions of Android Preview I could: 

* Tap on the tabs-button
* Tap on on of these websites
Done with two clicks. Compared with the old Firefox this made slightly less sense, since the space taken by the "top websites" could also have been used for open tabs. Nonetheless, it was still reasonably quick and convenient. 

In more recent versions of Firefox Nightly however, another step was introduced to open one of these frequently used websites. The same applies to opening a new tab in general. This too now involves an extra step. 

**Proposal**
I would propose to give users the option to open "top websites" immediately after tapping on the address bar. "Top websites" should be shown between the address bar and the keyboard. Focus should be an the address bar and once the user starts typing, the "top websites" should go away and instead search suggestions should pop up. 

**Alternative proposal**
Alternatively long press on the tabs button should no longer show a menu, but take the user directly to a new tab. At the moment this menu shows 3 options: 

* close current tab. ~~> this is something that can already be done in two clicks using the regular tabs~~screen. Doing this via a menu saves no time or effort and could thus painlessly be removed from the menu. 
* open new tab -> in my proposal this would immediately pop up, so one click less
* open new private tab ~~> this could be accessed via the the regular tabs~~screen

Please help us to keep the amount of hand / finger movement to a minimum.
